include/exadg/incompressible_navier_stokes/spatial_discretization/spatial_operator_base.cpp

```diff
@@ -1378,10 +1381,11 @@ SpatialOperatorBase<dim, Number>::evaluate_velocity_divergence_term(VectorType &
 
 template<int dim, typename Number>
 void
-SpatialOperatorBase<dim, Number>::update_turbulence_model(VectorType const & velocity)
+SpatialOperatorBase<dim, Number>::update_viscosity(VectorType const & velocity) const
 {
-  // calculate turbulent viscosity locally in each cell and face quadrature point
-  turbulence_model.calculate_turbulent_viscosity(velocity);
+  // calculate apparent viscosity for LES turbulence and/or generalized Newtonian models in each
```

Comment:
I think I would skip this comment to avoid having to update it again and again when adding other types of variable viscosity.